### PR TITLE
New dockerstuff

### PR DESCRIPTION
**Pull request by JosiahRegencia**
Used docker-compose run savant bash test.sh
as the command to run tests.

Build passed in 

https://travis-ci.org/Doveps/mono/builds/342182114

### CODE REVIEW

**Comment by greenmoss** [comment]
Please resolve the conflicts

**Comment by JosiahRegencia** [comment]
Did you mean in codeclimate?

**Comment by greenmoss** [comment]
Check out the "resolve conflicts" button. It means there was code committed on master, then conflicting code committed on this branch:

https://github.com/Doveps/mono/pull/97/conflicts

**Comment by greenmoss** [comment]
I did a deeper dive into this, and I think there are some problems with the `Dockerfile`. I detailed next steps in #99. Please take care of #99 first, then revisit this PR.

File: docker-compose.yml [hunk 1]
```diff
@@ -4,14 +4,18 @@ services:
     image: postgres
```
Comment (zerstoeren) [inline docker-compose.yml:4]: do we have our own docker file for this or are we using a docker file from somewhere else?

Comment (JosiahRegencia) [reply docker-compose.yml:4]: we have our own

**Comment by greenmoss** [comment]
I think this is no longer needed. Let me know if you want to keep it open.

**Pull request closed without merging**